src/scenes/Partners/Detail/Tabs/Projects/PartnerDetailProjectsTable.tsx

```diff
@@ -0,0 +1,135 @@
+import { Shield as ShieldIcon } from '@mui/icons-material';
+import { useTheme } from '@mui/material/styles';
+import { DataGrid, DataGridProps, GridRowParams } from '@mui/x-data-grid';
+import { useNavigate } from 'react-router-dom';
+import { ProjectStatusLabels } from '~/api/schema/enumLists';
+import { PartnerDetailsFragment } from '../../PartnerDetail.graphql';
+import { PartnerDetailProjectsTableListItemFragment } from './PartnerDetailProjectsTable.graphql';
+
+export interface PartnerDetailProjectsTableProps {
+  projects: readonly PartnerDetailProjectsTableListItemFragment[];
+  GridProps?: Omit<DataGridProps<RowData>, 'columns' | 'isCellEditable'>;
+}
+
+export interface RowData {
+  id: string;
+  label: string;
+  data: PartnerDetailsFragment[];
+}
+
+export const PartnerDetailProjectsTable = ({
+  projects,
+}: PartnerDetailProjectsTableProps) => {
+  const navigate = useNavigate();
+  const theme = useTheme();
+
+  const handleRowClick = (params: GridRowParams) => {
+    navigate(`/projects/${params.row.id}`);
+  };
+
+  return (
+    <DataGrid<PartnerDetailProjectsTableListItemFragment>
+      density="compact"
+      initialState={{
+        sorting: {
+          sortModel: [{ field: 'name', sort: 'desc' }],
+        },
+      }}
+      disableColumnMenu
+      disableSelectionOnClick
+      autoHeight
+      hideFooter={true}
+      showCellRightBorder={false}
+      rowSpacingType="margin"
+      sx={{
+        border: 'none',
+        '& .MuiDataGrid-columnSeparator': {
+          display: 'none',
+        },
+        '& .MuiDataGrid-columnHeaders': {
+          borderBottom: 'none',
+        },
+        '& .MuiDataGrid-columnHeaderTitle': {
+          fontWeight: 'bold',
+        },
+        '& .MuiDataGrid-cell': {
+          borderBottom: 'none',
+        },
+        '& .MuiDataGrid-row:hover': { cursor: 'pointer' },
+        '& .MuiDataGrid-cell, & .MuiDataGrid-columnHeader': {
+          '&:focus, &:focus-within': { outline: 'none' },
+        },
+        '& .MuiDataGrid-columnHeader:nth-last-of-type(-n+2) .MuiDataGrid-columnSeparator--sideRight':
```

Comment:
This is to remove the last column separator. On the table you copied it from we have an actions column as the last column. So there we are hiding the last two. But this table does not have an actions column.
```suggestion
        '& .MuiDataGrid-columnHeader:nth-last-of-type(-n+1) .MuiDataGrid-columnSeparator--sideRight':
```
This could probably be put in the `theme/overrides` file since we'd always want this.